core/merge.js

```diff
@@ -305,16 +306,18 @@ class Merge {
     metadata.markSide(side, doc, folder)
```

Comment:
Shouldn't we mark the sides after the id conflict detection as well? This is what we do for file moves.